nmigen_soc/csr/wishbone.py

```diff
@@ -90,5 +90,6 @@ def segment(index):
 
```

Comment:
Do you still need to clear ack, above? It seems to have become redundant.

Reply:
Thanks, that does look redundant. Updated the patch